src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+        action="`https://mate-academy-form-lesson.herokuapp.com/create-application`"
```

Comment:
Leave only one pair of brackets
```suggestion
        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
```

Reply:
👌🏻